tests/queries/test_explain.py

```diff
@@ -90,6 +90,8 @@ def test_postgres_options(self):
         ]
         if connection.features.is_postgresql_16:
             test_options.append({"generic_plan": True})
+        if connection.features.is_postgresql_17:
+            test_options.append({"memory": True, "serialize": True})
```

Comment:
As far as I'm aware, this options can be used separately.
```suggestion
        if connection.features.is_postgresql_17:
            test_options.append({"memory": True})
            test_options.append({"serialize": True})
```